xbmc/cores/RetroPlayer/RetroPlayer.cpp

```diff
@@ -447,6 +433,8 @@ void CRetroPlayer::FrameMove()
       break;
     }
     }
+
+    m_processInfo->SetPlayTimes(0, GetTime(), 0, GetTotalTime());
```

Comment:
be aware that CProcessInfo is private to VP. The more you use it, the more you'll have to change some day.